gamemodes/ix_cotz/plugins/background/sh_definitions.lua

```diff
@@ -152,4 +153,39 @@ ix.backgrounds["druggie"] = {
 		--client:SetRunSpeed(400) -- These things dont actually work, they're overridden by the stamina plugin almost instantly
 		--client:SetWalkSpeed(200)
 	end,
-}
\ No newline at end of file
+}
+
+ix.backgrounds["addict_light"] = {
```

Comment:
good idea with a combination of addictions - but we should maybe add a good starting gun or extra rubles for this guy since he is addicted to two things?